pkgs/top-level/python-packages.nix

```diff
@@ -3846,36 +3850,17 @@ in {
     };
   };
 
+  colorama = callPackage ../development/python-modules/colorama { };
 
-  colorama = buildPythonPackage rec {
-    name = "colorama-${version}";
+  colorama37 = self.colorama.overrideAttrs (old: rec {
```

Comment:
Changes in colorama have indeed caused breakage in the past. Let's leave such an optimization to the awscli maintainer.